sections/queueing_model/blocking_time/main.tex

```diff
@@ -0,0 +1,148 @@
+(\ref{eq:general_blocking_time_at_each_state}) can be solved by considering the 
+linear system \(Zx=y\). 
+Assuming \(i\) and \(j\) represent states \((u_i, v_i), (u_j, v_j) \in S_b\) 
+then \(Z_{ij}\) is given by:
+
+\begin{equation}\label{eq:general_mapping_function_of_blocking_matrix}
+    Z_{ij} = 
+    \begin{cases}
+        p_a, & \textbf{if } j = i + 1 \textbf{ and } v_i \neq N \\
+        \\
+        p_s, & \textbf{if } j = i - 1 \textbf{ and } v_i \neq N, v_i \neq T \\
+             & \textbf{or } j = i - N + T \textbf{ and } u_i \geq 2,\,v_i = T \\
+        \\
+        1, & \textbf{if } j = i - 1 \textbf{ and } v_i = N \\
+        \\
+        -1, & \textbf{if } i = j \\
+        \\
+        0, & \textbf{otherwise} \\
+    \end{cases}
+\end{equation}
+
+Equation~(\ref{eq:general_algebaric_approach_blocking_time}) shows this.
+\begin{equation}\label{eq:general_algebaric_approach_blocking_time}
+    \scalebox{0.73}{\(
+        Z = 
+        \begin{pmatrix}
+            -1 & p_a & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T)
+            p_s & -1 & p_a & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+1)
+            0 & p_s & -1 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+2)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 1 & -1 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,N)
+            p_s & 0 & 0 & \dots & 0 & 0 & -1 & p_a & 0 & \dots & 0 & 0 \\ %(2,T)
+            0 & 0 & 0 & \dots & 0 & 0 & p_s & -1 & p_a & \dots & 0 & 0 \\ %(2,T+1)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 1 & -1 \\ %(M,N)
+        \end{pmatrix},
+        x = 
+        \begin{pmatrix}
+            b(1,T) \\
+            b(1,T+1) \\
+            b(1,T+2) \\
+            \vdots \\
+            b(1,N) \\
+            b(2,T) \\
+            b(2,T+1) \\
+            \vdots \\
+            b(M,N) \\
+        \end{pmatrix}, 
+        y= 
+        \begin{pmatrix}
+            -c(1,T) \\
+            -c(1,T+1) \\
+            -c(1,T+2) \\
+            \vdots \\
+            -c(1,N) \\
+            -c(2,T) \\
+            -c(2,T+1) \\
+            \vdots \\
+            -c(M,N) \\
+        \end{pmatrix}
+    \)}
+\end{equation}
+
+
+Additional details on the blocking time formula 
+(\ref{eq:algebraic_blocking_time}) can be found in appendix 
+\ref{sec:appendix_mean_blocking}. 
+
+\subsubsection{Example}
+Figure \ref{fig:markov_vs_des_blocking_time_comparison} illustrates a comparison 
+between the formulas that arise from the Markov chain model and the equivalent 
+values of the blocking time extracted from discrete event simulation
+(appendix~\ref{sec:appendix_des}).
+The blocking time is calculated using both methods for a range of values of
+\(\lambda_2\) 
```

Comment:
You have not mentioned the 100 times repetition here?

Reply:
Indeed I have not. Must have missed it